### DISCUSSION TITLE
Problema Actividad 4 - Grupo 3

### DISCUSSION THREAD

**Discussion by renatacatelli** [frlp-utn-ingsoft/catedra#152] [Q&A]
Hola, buen día.

Estoy teniendo un problema cuando quiero probar los tests que hice para en mi caso, el refactor de validar que el numero de telefono del cliente sea un entero.

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/478f3a3b-9e78-4561-a578-7e5c8a5bb785)
 
Me muestra eso en consola. Como que salen OK pero me tira esos mensajes de error como que no los encuentra en el codigo, creo yo. Pero es raro porque en models los puse para que coincidan:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/e6e670b4-78df-4643-b802-01df29041a53)
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/5ffbff78-56fe-46d5-8460-ed28e024d7b0)
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/fa5c7228-5ae4-4bf4-b1d0-0099ed05ea47)

Desde ya gracias!!

**Comment by RodrigoJacznik** [comment]
Buenas, fijate que creo que te quedo un print dando vueltas.

**Reply by renatacatelli** [reply]
Gracias!! puede ser que era porque estaba usando el 'log' ? porque lo estuve usando en un momento para probar unos errores. Saqué eso y no me lo muestra más:

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/103262589/5ccc9544-13a7-4b4b-b9c2-5996da34714f)

**Comment by RodrigoJacznik** [comment]
Si, puede ser log tambien. Genial que lo solucionaste 👍
